### ISSUE TITLE
Make support channel more discoverable.

### ISSUE THREAD

**Issue by benjamreynolds** [travis-ci/travis-ci#8598]
I have spent a while searching around trying to find a support channel for TravisCI, but I was unable to find anything, hence I am filing this issue. First and foremost, if there is a better channel for something like this, could someone point me in that direction?

My actual issue is that TravisCI is flagging my builds as abuse and they are not running. See caskroom/homebrew-cask#39723 for an example. I am trying to figure out why this is happening so that I can fix whatever it is that is causing this.

**Comment by cotsog** [comment]
@benjamreynolds: Thanks for reaching out and I'm sorry for the troubles.

Our system automatically flagged your account for possible abuse, but after further review, we were able to clear that up. You should be all set to build as normal, though please feel free to contact us at `support [at] travis-ci [dot] com` should you have any questions or concerns arise.

**Comment by benjamreynolds** [comment]
Thanks! might I suggest making that support email more widely visible online? As far as I can tell it does not show up anywhere in the first page of results when I search "travis ci support" on google.

**Comment by BanzaiMan** [comment]
> In addition, I didnt find anything obvious browsing around on travis-ci.org

How can we make the "email" link in the bottom footer more discoverable for you?

![travis_ci_-_test_and_deploy_your_code_with_confidence](https://user-images.githubusercontent.com/25666/31676066-7501654e-b334-11e7-9398-96ce495d7f7a.png)

**Comment by BanzaiMan** [comment]
Was https://github.com/travis-ci/travis-ci/issues/ more discoverable for you? Where did you find this?

**Comment by benjamreynolds** [comment]
Seems pretty obvious now that I see the email link under the "help" subheading. Basically I was doing alot of googling things like "travis ci support", "travis ci contact", etc. I guess "help" just didnt occur to me as a comparable thing. more of a fault on my end than your in retrospect. 😞 

Also, once I created an account and logged in (I never had a reason to have an account so I just did so yesterday, as I just contribute to open source, I dont really have any of my own open-source projects) I see that there is a notification center in your nav bar that told me I my account had been flagged and had a mail-to link to contact support.

Ill Close, I think this was more me than you guys.
